### DISCUSSION TITLE
Error in "post_fp_cp2k" link, ask for help.

### DISCUSSION THREAD

**Discussion by bizhisheng** [deepmodeling/dpgen#1380] [Q&A]
Hello, everybody!

I recently encountered a problem during the operation of DPGEN, and I still can't solve it. So let's ask the professionals.

In the process of running the CH4 case given by DPGEN official website, it is very smooth. When I changed the system to TiO2, the following error appeared in the "**post_fp_cp2k**" link. (**nohup.out** file)

The difference between CH4 and TiO2 cases is that TiO2 takes **2000K temperature**.

Thank you for reading, and look forward to having experts provide valuable suggestions and opinions!
[nohup.txt](https://github.com/deepmodeling/dpgen/files/12816042/nohup.txt)

**Comment by njzjz** [comment]
@robinzyb Could you take a look?

**Comment by robinzyb** [comment]
I found the dpdata didn't find cell in your dpgen output.  this is most probably caused by empty appending. you can try to write a loop by yourself using dpdata to read cp2k data in 02.fp files to find out the output with problem.

**Comment by 12jscvb** [comment]
Hello, I have the same problem as you. Have you solved your problem?

**Comment by bizhisheng** [comment]
> @robinzyb Could you take a look?

Professor Zhuang
Hello, you mean to let me use the "**cp2kdata**" toolkit, right? Excuse me, how can I embed **cp2kdata** into **dpgen**?

**Comment by robinzyb** [comment]
I didn't mean that. I am not sure if some calculation in fp.02 haven't done or read properly. Could you upload the whole fp.02?

**Comment by bizhisheng** [comment]
> I didn't mean that. I am not sure if some calculation in fp.02 haven't done or read properly. Could you upload the whole fp.02?

Professor zhuang
Hello, this is **02.fp** related documents for your reference.
[02.fp.zip](https://github.com/deepmodeling/dpgen/files/13204179/02.fp.zip)

**Comment by robinzyb** [comment]
because none of your cp2k calculation is converged. 
```shell
tsf-500-wpa-7-033:robinzhuang(02.fp)$ grep NOT task.000.00000*/output*
task.000.000000/output: *** WARNING in qs_scf.F:542 :: SCF run NOT converged ***
task.000.000001/output: *** WARNING in qs_scf.F:542 :: SCF run NOT converged ***
task.000.000002/output: *** WARNING in qs_scf.F:542 :: SCF run NOT converged ***
task.000.000003/output: *** WARNING in qs_scf.F:542 :: SCF run NOT converged ***
task.000.000004/output: *** WARNING in qs_scf.F:542 :: SCF run NOT converged ***
task.000.000005/output: *** WARNING in qs_scf.F:542 :: SCF run NOT converged ***
task.000.000006/output: *** WARNING in qs_scf.F:542 :: SCF run NOT converged ***
task.000.000007/output: *** WARNING in qs_scf.F:542 :: SCF run NOT converged ***
task.000.000008/output: *** WARNING in qs_scf.F:542 :: SCF run NOT converged ***
task.000.000009/output: *** WARNING in qs_scf.F:542 :: SCF run NOT converged ***
```
I will add error message for this, later.

**Comment by robinzyb** [comment]
This problem should be fixed https://github.com/deepmodeling/dpgen/pull/1245, which is now in the devel branch. But it hasn't merged to main/master branch. To avoid the problem, try install DP-GEN In devel branch. @bizhisheng

**Comment by robinzyb** [comment]
One more, it is abnormal that all cp2k jobs are failed. check the setups

**Comment by bizhisheng** [comment]
Professor Zhuang
Thank you for checking. Your suggestion is very important to me, and I will carefully check the related mistakes. I wish you success in your work and good health!

**Comment by bizhisheng** [comment]
> Hello, I have the same problem as you. Have you solved your problem?

Hello, I haven't solved the related problem for the time being, but you can read the comments in the follow-up. Maybe it will help you.
